R/pkg/R/DataFrame.R

```diff
@@ -1231,6 +1305,22 @@ setMethod("unionAll",
             dataFrame(unioned)
           })
 
+#' @title Union two or more DataFrames
+#
+#' @description Returns a new DataFrame containing rows of all parameters.
+#
+#' @rdname rbind
+#' @aliases unionAll
+  setMethod("rbind",
```

Comment:
theres an extra line of space before this `setMethod`, but I can fix this up during merge